LICENSE

```diff
@@ -1,6 +1,4 @@
-BSD 3-Clause License
-
-Copyright (c) 2020, QuTiP
+Copyright (c) 2011 to 2021 inclusive, QuTiP developers and contributors.
```

Comment:
Since qutip-qip is new, I'm not sure you can assert copyright from past years. Also, since a lot of code is migrated from QuTiP main, I think to be in compliance with the QuTiP licence, we may need a statement of attribution about that as well.

Reply:
Yeah, maybe this can only be `Copyright (c) 2020 to 2021 inclusive, QuTiP developers and contributors.`

Reply:
But indeed I'm not sure if we need to keep a different licence for those files migrated from `qutip` and those created new in `qutip-qip`.

Reply:
Since this is a QuTiP family package and this is a "fork" of a subset of qutip, I think it can keep the "2011 to 2021 inclusive" and the "QuTiP developers and contributors".

If the contributors were not the same we would write:

```
Copyright (c) 2021, QuTiP QIP developers and contributors.
Copyright (c) 2011 to 2021 inclusive, QuTiP developers and contributors.
```

Because we need to retain the old copyright years and owners and add the new ones. Since they're the same, we can just merge them.